src/lib.rs

```diff
@@ -280,14 +300,28 @@ where
         Ok(())
     }
 
+    /// Turn data whitening on / off.
+    pub fn white_data_enable(&mut self, enable: bool) -> Result<(), Error<SpiE>> {
+        self.0.modify_register(Config::PKTCTRL0, |r| {
+            PKTCTRL0(r).modify().white_data(enable as u8).bits()
+        })?;
+        Ok(())
+    }
```

Comment:
Nice, realize that it's time to trim the ugly set_defaults function a bit. Close to getting rid of it now. I perhaps should have mentioned its (in)stability in the doc comment for it :D